pkg/platform/provider/baremetal/cluster/create.go

```diff
@@ -1660,8 +1660,8 @@ func (p *Provider) EnsureCheckAnywhereSubscription(ctx context.Context, c *v1.Cl
 	if err != nil {
 		return err
 	}
-	for i, feed := range sub.Spec.Feeds {
-		_ = wait.PollImmediate(5*time.Second, 1*time.Minute, func() (bool, error) {
+	_ = wait.PollImmediate(5*time.Second, 10*time.Minute, func() (bool, error) {
```

Comment:
The inspection frequency is too high， recommend 30s

Reply:
30s is a little bit long, 15s?

Reply:
15s ok for me